languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -22925,7 +22925,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <example>Der Vortrag wurde präsentiert von Michael Krass.</example>
                 <example>Prof. Krass leitet heute das Seminar.</example>
             </rule>
-            <rule id="KOMPLETT_STIL" name="komplett (völlig, vollkommen)" default="temp_off" tags="picky">
+            <rule id="KOMPLETT_STIL" name="komplett (völlig, vollkommen)" tags="picky">
```

Comment:
Bin hier [beim Diff](https://internal1.languagetool.org/regression-tests/via-http/2022-03-28/de-DE_full/result_grammar_KOMPLETT_STIL[1].html) noch sehr skeptisch.
Die Regel würde naturgemäß sehr oft matchen. Sowas wie "komplett eingrichtet" erscheint mir nicht ugs., ich finde das in dem Fall sogar am geläufigsten. Bist du sicher, dass das ugs. ist? Auch "komplett ausgestattet"
"Oder komplett alles?" ist nochmal ein eigener Fall...

Am besten wäre es wohl, die Regel irgendwie zu bändigen.